src/preprocessing/particle_packing/nhs_faces.jl

```diff
@@ -24,6 +24,8 @@ function faces_in_cell(cell, neighborhood_search)
     return PointNeighbors.points_in_cell(cell, neighborhood_search)
 end
 
+@inline eachneighbor(coords, nhs::TrivialNeighborhoodSearch) = nhs.eachpoint
```

Comment:
```suggestion
@inline eachneighbor(coords, nhs::TrivialNeighborhoodSearch) = PointNeighbors.eachneighbor(coords, nhs)
```
Don't use internals.